hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/MockRatisPipelineProvider.java

```diff
@@ -37,4 +37,9 @@ public MockRatisPipelineProvider(NodeManager nodeManager,
   protected void initializePipeline(Pipeline pipeline) throws IOException {
     // do nothing as the datanodes do not exists
   }
+
+  @Override
+  public void shutdown() {
+
```

Comment:
NIT - Can we add a comment like // do nothing?

Reply:
Done